test/parallel/test-http2-misbehaving-multiplex.js

```diff
@@ -11,35 +11,18 @@ const h2 = require('http2');
 const net = require('net');
 const { NghttpError } = require('internal/http2/util');
 const h2test = require('../common/http2');
-let client;
 
 const server = h2.createServer();
-let gotFirstStreamId1;
 server.on('stream', common.mustCall((stream) => {
   stream.respond();
   stream.end('ok');
 
-  // Http2Server should be fast enough to respond to and close
-  // the first streams with ID 1 and ID 3 without errors.
-
-  // Test for errors in 'close' event to ensure no errors on some streams.
-  stream.on('error', () => {});
-  stream.on('close', (err) => {
-    if (stream.id === 1) {
-      if (gotFirstStreamId1) {
```

Comment:
If this branch was taken, the test would fail because there is nothing that calls the function returned by `common.expectsError()` below.